jonathan-tschida/travel-tracker
# Iteration 1

#### What's this PR do?
Creates a log in screen and lets the user log in.
#### Where should the reviewer start?
What needs to be looked at
#### How should this be manually tested?
Try to log in with incorrect usernames and passwords.
The password for all users is:
 - 'travel2020'
Valid user names are:
 - 'agency'
 - 'traveler1' up to 'traveler50'
Invalid user names should be:
 - 'traveler0'
 - 'traveler51'
 - 'traveler123'
 - 'travelers49'
 - 'Traveler23'
 - 'sexytraveler23'
#### Questions:
- Is the log in pop up too far down the page?
- The welcome messages are just placeholders for dashboards.
